### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by rafaelbco**
* Convert double to single quotes in Python code
* Remove Extensions/Install.py because it is not needed for Plone 4+

### CODE REVIEW

File: collective/logbook/Extensions/Install.py [hunk 1]
```diff
@@ -1,47 +0,0 @@
...
-
-def uninstall_storages(portal):
-    annotations = IAnnotations(portal)
-    if annotations.get(STORAGE_KEY):
-        LOGGER.info("*** UNINSTALL collective.logbook Logstorage ***")
-        del annotations[STORAGE_KEY]
-
-    if annotations.get(INDEX_KEY):
-        LOGGER.info("*** UNINSTALL collective.logbook Indexstorage ***")
-        del annotations[INDEX_KEY]
```
Comment (ramonski) [inline collective/logbook/Extensions/Install.py:47]: Hmm, I knew that this approach was old-school, but didn't remember if the code was still executed in Plone 4+.
Would you mind to add a `setuphandlers.py` in the root with that code and link it in the GS profile? I would like to have `collective.logbook` cleanly removable. Probably we should to remove the controlpanel configlet then as well.

Comment (rafaelbco) [reply collective/logbook/Extensions/Install.py:47]: I'm on it.

**Comment by rafaelbco** [comment]
Now the add-on installs and uninstalls cleanly.

**Comment by ramonski** [comment]
Going to release as soon as possible. Thanks for your efforts and contribution Rafael!

**Comment by ramonski** [comment]
Released 1.0.0: https://pypi.org/project/collective.logbook

**Pull request merged**